dev/TwoPaneView/TwoPaneView.cpp

```diff
@@ -26,6 +26,7 @@ TwoPaneView::TwoPaneView()
     SetDefaultStyleKey(this);
 
     SizeChanged({ this, &TwoPaneView::OnSizeChanged });
+    winrt::Window::Current().SizeChanged({ this, &TwoPaneView::OnWindowSizeChanged });
```

Comment:
>winrt::Window::Current().SizeChanged({ this, &TwoPaneView::OnWindowSizeChanged }); [](start = 4, length = 82)

I think this is not island friendly, I think we need to use appwindow.

Reply:
The dual-screen part of TwoPaneView doesn't work in islands, because we are tied to app view. Comp/shell work needs to be done in order for us to be able to be island-friendly.